QUEST_LV_0100.tsv

```diff
@@ -596,27 +596,27 @@ QUEST_LV_0100_20150317_000595	$Role of the Guardians
 QUEST_LV_0100_20150317_000596	$By recycling the power supply of the Guardians that forgot their mission, we can return them to be part of the Royal Mausoleum.
 QUEST_LV_0100_20150317_000597	$Guardian's Stone Statue
 QUEST_LV_0100_20150317_000598	$The Guardians are corrupted with evil energy.{nl}Destroy the corrupted Guardians to stop its spreading.
-QUEST_LV_0100_20150317_000599	$The demons are trying to take over the roots of the tomb.{nl}If all controlling magics get corrupted, we can't protect the tomb anymore.
+QUEST_LV_0100_20150317_000599	$The demons are trying to take over the roots of the Royal Mausoleum.{nl}If all controlling magic becomes corrupted, we can't protect the Royal Mausoleum anymore.
 QUEST_LV_0100_20150317_000600	$The Guardians are being corrupted like a plague.{nl}We need to look for a special way.
 QUEST_LV_0100_20150317_000601	$We need to destroy the corrupted Guardians that are in sleep.{nl}First, collect condensed magic in this jar.
 QUEST_LV_0100_20150317_000602	$I pray to the Goddess for those sleeping Guardians.
 QUEST_LV_0100_20150317_000603	$We should destroy those Guardians before they wake up to listen to demons' commands.
-QUEST_LV_0100_20150317_000604	$Destroy the Guardians using the condensed magic.{nl}If something interferes, destroy it as well.
+QUEST_LV_0100_20150317_000604	$Destroy the Guardians using the condensed magic source.{nl}If something interferes, destroy it as well.
 QUEST_LV_0100_20150317_000605	You should destroy them before they wake up.
 QUEST_LV_0100_20150317_000606	$It's very sad to listen to the commands of demons as the Guardians of the tomb.
 QUEST_LV_0100_20150317_000607	$This evil is very real.{nl}It could possibly reach the deepest parts of the Royal Mausoleum.
 QUEST_LV_0100_20150317_000608	Royal Mausoleum Blueprint
-QUEST_LV_0100_20150317_000609	$If a force of evil is close to the Goddess' revelation, Guardians are programmed to destroy the Royal Mausoleum.
-QUEST_LV_0100_20150317_000610	$The evil man is getting close to the Goddess' revelation.{nl}But he will never get the revelation.{nl}
+QUEST_LV_0100_20150317_000609	$If the evil presence is close to the Goddess' revelation, the Guardians are programmed to destroy the Royal Mausoleum.
```

Comment:
Could be:
"If the evil presence comes close to"
or
"If the evil presence gets too close to"